templates/base/src/routes/index.js

```diff
@@ -7,3 +7,6 @@ export const NextRouterWrapper = ({ pathName, as, options = {} }) => {
 };
 
 export const getBasePath = (route) => `${BASE_PATH}${route}`;
+
+export const HOMEPAGE = '/';
```

Comment:
@pras9 These should be maintained in a separate file, I think there is a routeConfig.js

Reply:
@vinodloha PR comments resolved, PR updated